fastlane/swift/Fastlane.swift

```diff
@@ -3250,7 +3250,7 @@ func slather(buildDirectory: String? = nil,
              verbose: Bool? = nil,
              useBundleExec: Bool = false,
              binaryBasename: Bool = false,
-             binaryFile: Bool = false,
+             binaryFile: String?,
```

Comment:
This unfortunately will be overwritten as the whole file is generated from slather.rb.

My guess: is_string: true, in line 259 of slather.rb would result in what you want to have here. (But really just a guess, I don't even know how this is generated)

Reply:
The file is checked in though so I figure at least the checked in version should match the generated one.  Is this ok or would it be preferred this file stay untouched?

Reply:
Looking at the file history there, it is generated during the release process. So you could change it here in the PR, but it would be overwritten on release anyway.

Reply:
Got it!  I'll put Fastlane.swift back to how it is.  Sorry, I must of missed the documentation on this.

Reply:
Very possible that doesn'T exist, as all the Swift stuff is in beta and only semi supported.

Reply:
Groovy!  Let me know if anything else is needed.